packages/neon-invoker/src/index.ts

```diff
@@ -2,25 +2,33 @@ import { ContractInvocationMulti, Signer, Neo3Invoker, Arg, InvokeResult } from
 import { tx, u, rpc, sc, experimental } from '@cityofzion/neon-js'
 import * as Neon from '@cityofzion/neon-core'
 import { wallet } from '@cityofzion/neon-core'
+import { CommonConfig } from '@cityofzion/neon-js/lib/experimental/types'
 
 export type RpcConfig = {
   rpcAddress: string
   networkMagic: number
 }
 
+export interface BuildTransaction {
```

Comment:
I think this type name is strange, we have 2 options:
1) rename to BuildTransactionInput
2) not use a type and receive them as param